app/app.go

```diff
@@ -707,6 +707,10 @@ func (app *App) InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci.Res
 	if err := tmjson.Unmarshal(req.AppStateBytes, &genesisState); err != nil {
 		panic(err)
 	}
+
+	// set module version map
+	// https://docs.cosmos.network/main/core/upgrade#genesis-state
+	app.UpgradeKeeper.SetModuleVersionMap(ctx, app.mm.GetVersionMap())
```

Comment:
This should have been added in the previous upgrade(v2.0.5).
It's for set module version map when initializing chain.
No effect on this upgrade.